tests/ops/op_math/test_controlled_decompositions.py

```diff
@@ -294,88 +201,30 @@ def test_trivial_ops_in_decomposition(self):
                     qml.RZ(np.pi / 2, wires=0),
                     qml.RY(np.pi / 2, wires=0),
                     qml.CNOT(wires=[1, 0]),
                     qml.RY(-np.pi / 2, wires=0),
                     qml.RZ(-2 * np.pi, wires=0),
                     qml.CNOT(wires=[1, 0]),
                     qml.RZ(3 * np.pi / 2, wires=0),
+                    qml.ctrl(qml.GlobalPhase(phi=-np.pi / 2), control=1),
                 ],
             ),
         ],
     )
     def test_composite_ops(self, composite_op, want_decomp):
         """Test that ZYZ decomposition is used for composite operators."""
         have_decomp = ctrl_decomp_zyz(composite_op, 1)
-        assert have_decomp == want_decomp
-
-    @pytest.mark.parametrize("test_expand", [False, True])
-    def test_zyz_decomp_no_control_values(self, test_expand):
-        """Test that the ZYZ decomposition is used for single qubit target operations
-        when other decompositions aren't available."""
-
-        base = qml.QubitUnitary(
-            np.array(
-                [
-                    [1, 1],
-                    [-1, 1],
-                ]
-            )
-            * 2**-0.5,
-            wires="a",
-        )
-        op = Controlled(base, (0,))
-
-        assert op.has_decomposition
-        decomp = (
-            qml.tape.QuantumScript(op.decomposition()).expand().circuit
-            if test_expand
-            else op.decomposition()[0].decomposition()
-        )
-        expected = qml.ops.ctrl_decomp_zyz(base, (0,))  # pylint:disable=no-member
-        assert equal_list(decomp, expected)
-
-    @pytest.mark.parametrize("test_expand", [False, True])
-    def test_zyz_decomp_control_values(self, test_expand):
-        """Test that the ZYZ decomposition is used for single qubit target operations
-        when other decompositions aren't available and control values are present."""
-        # pylint:disable=no-member
-        base = qml.QubitUnitary(
-            np.array(
-                [
-                    [1, 1],
-                    [-1, 1],
-                ]
-            )
-            * 2**-0.5,
-            wires="a",
-        )
-        op = Controlled(base, (0,), control_values=[False])
-
-        assert op.has_decomposition
-        decomp = (
-            qml.tape.QuantumScript(op.decomposition()).circuit
-            if test_expand
-            else op.decomposition()
-        )
-        assert len(decomp) == 3
-        qml.assert_equal(qml.PauliX(0), decomp[0])
-        qml.assert_equal(qml.PauliX(0), decomp[-1])
-        decomp = decomp[1]
-        decomp = (
-            qml.tape.QuantumScript(decomp.decomposition()).circuit
-            if test_expand
-            else decomp.decomposition()
-        )
-        expected = qml.ops.ctrl_decomp_zyz(base, (0,))
-        assert equal_list(decomp, expected)
+        assert len(have_decomp) == len(want_decomp)
+        for actual, expected in zip(have_decomp, want_decomp):
```

Comment:
Pretty sure this would do the same thing.
```suggestion
        for actual, expected in zip(have_decomp, want_decomp, strict=True):
```